rdwatch/core/views/server_status.py

```diff
@@ -40,6 +41,16 @@ def get_status(request: HttpRequest):
     hostname = socket.gethostname()
     ip = socket.gethostbyname(hostname)
 
+    parsed_version = Version.parse(api.version)
+
+    version = 'v'
+    version += '.'.join(str(v) for v in parsed_version.release)
+
+    # If this is a pre-release version, include the pre-release version string
+    post_num: int | None = parsed_version.post
+    if post_num is not None:
+        version += f'.post{post_num}+{parsed_version.local}'
```

Comment:
What's the difference between `api.version` and the string constructed here?

Reply:
When the current version is not a tagged release, they differ only slightly. For example, for the latest commit on this branch
```python
print(api.version)  # prints '1.10.2.post5.dev0+b074e39c'
print(version)  # prints 'v1.10.2.post5+b074e39c'
```

The main motivation behind this change is to make it look more user friendly when the curent version is a tagged release, which looks like this
```python
print(api.version)  # prints '1.12+b074e39c'
print(version)  # prints 'v1.12'
```